extensions/ql-vscode/src/view/variant-analysis/VariantAnalysisActions.tsx

```diff
@@ -9,13 +10,17 @@ export type VariantAnalysisActionsProps = {
   stopQueryDisabled?: boolean;
 
   showResultActions?: boolean;
+  onViewAutofixesClick: () => void;
   onCopyRepositoryListClick: () => void;
   onExportResultsClick: () => void;
+  viewAutofixesDisabled?: boolean;
   copyRepositoryListDisabled?: boolean;
   exportResultsDisabled?: boolean;
 
   hasSelectedRepositories?: boolean;
   hasFilteredRepositories?: boolean;
+
+  userSettings: UserSettings;
```

Comment:
Could this just be something like `showViewAutofixesButton` to make this slightly easier to test (since you won't need to construct the full user settings)?

Reply:
Updated in https://github.com/github/vscode-codeql/pull/4065/commits/04a858affa7fc4b6fe49362b8b7bfa4895fa3794.